modules/xmpp/JingleSessionPC.js

```diff
@@ -1668,8 +1668,10 @@ export default class JingleSessionPC extends JingleSession {
 
             // NOTE the code below assumes that no more than 1 video track
             // can be added to the peer connection.
-            // Transition from no video to video (possibly screen sharing)
-            if (!oldTrack && newTrack && newTrack.isVideoTrack()) {
+            // Transition from no video to video (unmute),
+            // transition from camera to desktop share
+            // or transition from one camera source to another.
+            if (newTrack && newTrack.isVideoTrack()) {
```

Comment:
can you remind me what `isVideoTrack` denotes?

Reply:
it tells whether it's an audio or a video track

Reply:
in that case @jallamsetty1 will this trigger on a camera change?

Reply:
yes, it will